recipe/meta.yaml

```diff
@@ -0,0 +1,118 @@
+{% set name = "conda-store" %}
+{% set version = "2024.1.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  # This is only needed here in-repo; in conda-forge, sources from PyPI
+  - path: ../
+
+build:
+  number: 0
+
+outputs:
+  - name: conda-store
+    version: {{ version }}
+    build:
+      noarch: python
+      script: python -m pip install ./conda-store -vv --no-deps --no-build-isolation --no-index
```

Comment:
Had to add extra flags here. pip was running _with_ build isolation due to https://github.com/conda/conda-build/issues/3993